File: projekat/src/app/entities/destinacija/destinacija.ts
import { NumberValueAccessor } from '@angular/forms';

export class Destinacija {
    id: number;
    nazivDestinacije: string
    datumVremeSletanja: string;
    datumVremePoletanja: string;
    vremePutovanja: string;
    duzinaPutovanja: string;
    brojPresedanja: number;
    lokacijaPresedanja: string;
    cenaKarte: string;
    airCompanyId: number;


    constructor(id:number, nazivDestinacije: string, datumVremeSletanja: string, datumVremePoletanja: string, vremePutovanja: string, duzinaPutovanja: string, brojPresedanja: number, lokacijaPresedanja: string, cenaKarte: string) {
        this.id = id;
        this.nazivDestinacije = nazivDestinacije;
        this.datumVremeSletanja = datumVremeSletanja;
        this.datumVremePoletanja = datumVremePoletanja;
        this.vremePutovanja = vremePutovanja;
        this.duzinaPutovanja = duzinaPutovanja;
        this.brojPresedanja = brojPresedanja;
        this.lokacijaPresedanja = lokacijaPresedanja;
        this.cenaKarte = cenaKarte;
    }

}

File: WebApplication1/WebApplication1/Models/Destinacija.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication1.Models
{
    public class Destinacija
    {
        [Key]
        public int Id { get; set; }

        public string NazivDestinacije { get; set; }

        public string datumVremeSletanja { get; set; }

        public string datumVremePoletanja { get; set; }

        public string vremePutovanja { get; set; }

        public string duzinaPutovanja { get; set; }

        public int brojPresedanja { get; set; }

        public string lokacijaPresedanja { get; set; }

        public string cenaKarte { get; set; }
    }
}
